aws-transfer-server/src/test/java/software/amazon/transfer/server/UpdateHandlerTest.java

```diff
@@ -205,6 +205,13 @@ public void stabilizationFailures() {
 
     private ProgressEvent<ResourceModel, CallbackContext> updateServerAndAssertStatus(
             ResourceHandlerRequest<ResourceModel> request, String postUpdateState, OperationStatus operationStatus) {
+        // Create a copy to avoid the update handler mutating the original.
```

Comment:
I initially made some changes to the Update tests here, and had to make this change as well. I've since undone those changes since I realised they were unnecessary. So, this change may now be unnecessary too, but I still think it's good safety to have it.